packages/medusa/src/services/price-list.ts

```diff
@@ -336,6 +345,28 @@ class PriceListService extends BaseService {
       return [productsWithPrices, count]
     })
   }
+
+  /**
+   * Add `currency_code` to an MA record if `region_id`is passed.
+   * @param prices - a list of PriceListPrice(Create/Update)Input records
+   * @return {Promise} updated `prices` list
+   */
+  private async addCurrencyFromRegion<
```

Comment:
I would rather set the visibility to protected instead to be open to extension :)